### PR TITLE
adding contributors to the operator

### PR DESCRIPTION
**Pull request by svanellewee**
Adding contributors https://github.com/yugabyte/yugabyte-operator/pull/6

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -87,3 +87,5 @@ This is a list of people who have contributed code to the [YugabyteDB](https://g
 * [eliahburns](https://github.com/eliahburns)
 * [bhavin192](https://github.com/bhavin192)
 * [graffido](https://github.com/graffido)
+* [svanellewee](https://github.com/svanellewee)
+* [nvulane](https://github.com/nvulane)
```
Comment (rkarthik007) [inline CONTRIBUTORS.md]: Hi @svanellewee ,

Would be good if @nvulane can send a separate PR with this entry. This way, the listing on github contributors page for yugabyte would also be accurate: https://github.com/yugabyte/yugabyte-db/graphs/contributors

Could you please modify the PR to just add your entry? Thanks!

Comment (svanellewee) [reply CONTRIBUTORS.md]: Sure thing, I'll ping him.

Comment (svanellewee) [reply CONTRIBUTORS.md]: Oh! I guess it makes  sense, means that he is also signing his contributor agreement.

**Comment by svanellewee** [comment]
thank you!
